css/length-indicator.css

```diff
@@ -1,30 +1,30 @@
-.indicator {
+.length-indicator__indicator {
   background: #ddd;
   width: 100%;
   height: 10px;
   opacity: 0.2;
 }
 
-.indicator.bad {
+.length-indicator__indicator--bad {
   background: red;
 }
-.indicator.ok {
+.length-indicator__indicator--ok {
   background: orange;
 }
-.indicator.good {
+.length-indicator__indicator--good {
   background: green;
 }
 
-.indicator-wrap {
+.length-indicator {
   display: flex;
   position: relative;
 }
 
-.active {
+.length-indicator__indicator.is-active {
```

Comment:
Why do we use here non-bem `is-active` instead of `length-indicator__indicator--is-active` as we are using for the good / ok / bad indication?

Reply:
(not that I'm complaining; I'm just wondering)

Reply:
That seems to be what https://www.drupal.org/docs/develop/standards/css/css-architecture-for-drupal-8#separate-concerns is saying for javascript added state. I'm no BEM expert...